lib/mailbox.c

```diff
@@ -78,7 +78,7 @@ mb_alloc_entry(struct mailbox *mb)
 {
 	void *obj = NULL;
 	int ret = rte_mempool_get(mb->pool, &obj);
-	if (ret == -ENOENT) {
+	if (ret < 0) {
 		G_LOG(ERR, "mailbox: not enough entries in the mempool\n");
```

Comment:
Given that the idea is to handle any error, we should update the error message and include the error code and `strerror(3)` of the error code.